Apply prescribed 3D ocean temperature field from a file

carolinevancalcar opened the pull request:
This merge contains the new option "prescribed" for choice_ocean_model_config. Using this option will read a file (give filename at filename_prescribed_oceanT_config) with a 3D ocean temperature field (name variable T_ocean). The temperature is kept constant over the run unless an ocean temperature inversion is turned on. The salinity is taken from the world ocean atlas (present day observed).

This option is for example useful if you want to do a simulation with a constant ocean temperature that is not present day observed. Or if you want to spinup with the inverted ocean temperature from another spinup as the starting point.

carolinevancalcar committed:
Add option to read a prescribed ocean temperature

The option "prescribed" for choice_ocean_model_config reads ocean
temperature from a file and keeps the temperature constant over
time. Salinity is taken from world ocean atlas.

carolinevancalcar committed:
Merge branch 'dev' of https://github.com/IMAU-paleo/IMAU-ICE into issue/oceantemperature

MScherrenberg reviewed (commented):
Looks good and clean, but my main suggestion would be to change the name "prescribed" to something that reflects it being 1) constant temperature and 2) having different sources for temperature and salinity. I think that would help prevent confusion in the future.

Additionally, I think it's a good idea to add this setting to the comments of choice_ocean_model_config in the configuration module.

carolinevancalcar committed:
Change name of option for choice_ocean_model

The option to prescribe ocean temperature was first called
"prescribed" and is now called "prescribed_oceanT" to
be more descriptive. Also added the option in the comment
in the configuration file.

carolinevancalcar commented:
> Looks good and clean, but my main suggestion would be to change the name "prescribed" to something that reflects it being 1) constant temperature and 2) having different sources for temperature and salinity. I think that would help prevent confusion in the future.
> 
> Additionally, I think it's a good idea to add this setting to the comments of choice_ocean_model_config in the configuration module.

Thanks @MScherrenberg. I changed the name to "prescribed_oceanT" and add it in the comment in the configuration file.

MScherrenberg reviewed (approved):
Top, en fijn ook dat je nog een oude aanpassing bij choice_climate_model_config had toegevoegd! Ik keur het goed
